### DISCUSSION TITLE
DS 算出来的md5不对

### DISCUSSION THREAD

**Discussion by Fantasy135** [UIGF-org/mihoyo-api-collect#66] [Q&A] [answered]
我手机抓包抓出的DS是
1720226815,125992,49fc7e532362e7d06da5db88850ee14c

![image](https://github.com/UIGF-org/mihoyo-api-collect/assets/57759754/16d09c79-571c-44f8-9f49-27ca75d8fc84)

是salt改变了吗

**Comment by Womsxd** [comment, marked as answer]
![image](https://github.com/UIGF-org/mihoyo-api-collect/assets/45663319/ab8b70f2-4db1-4662-9a80-c941575a11f3)

4e : salt=t0qEgfub6cvueAPgR5m9aQWWVciEer7v&t=1720226815&r=125992&b={"gids": "5"}&q=
49 : salt=t0qEgfub6cvueAPgR5m9aQWWVciEer7v&t=1720226815&r=125992&b={"gids":"5"}&q=

注意json里面的空格

**Comment by MiniGrayGay** [comment]
乐，空格大法好
